packages/cactus-plugin-keychain-google-sm/package-lock.json

```diff
@@ -0,0 +1,3040 @@
+{
```

Comment:
@jagpreetsinghsasan package-lock.json files are now obselete and need to be deleted since the yarn migration (happened today). Please delete this file from the commit (e..g delete the file, then commit that deletion).